index.js

```diff
@@ -23,10 +21,32 @@ const imagesArray = [
     {src: 'image/dog6.jpeg', alt: 'dog6'},
 ];
 
-const randomImage = (function () {
-    const randomNumber = Math.floor(Math.random() * imagesArray.length);
-    return imagesArray[randomNumber];
-}) ();
+let a = 0;
+
+prev.addEventListener('click', () => {
+    a--
+    if (a === imagesArray.length - 2) {
+        next.style.visibility = "visible";
+    }
+
+    img.setAttribute('src', imagesArray[a].src);
+    img.setAttribute('alt', imagesArray[a].alt);
+
+    if (a === 0 ) {
+        prev.style.visibility = 'hidden';
+    }
+}); 
+
+next.addEventListener('click', () => {
+    a++
+    if (a > 0) {
```

Comment:
```suggestion
    if (a < imagesArray.length - 1) {
```
мені здається що так буде трохи логічніше